### DISCUSSION TITLE
My account is lost

### DISCUSSION THREAD

**Discussion by riziqalbab** [community/community#137217] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

Hi all, I'm having a trouble, when I want to log into my main account it shows Incorrect username or password. After I checked, it turned out that my account was lost. Whether my account was hacked or what I don't know. Can anyone help me to recover my account?

My account name AlbabRiziq

**Comment by radar-blip** [comment, marked as answer]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Reply by riziqalbab** [reply]
Thanks, I'll try it

**Reply by queenofcorgis** [reply]
Hi @riziqalbab, 

Please note: the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
